### PR TITLE
Updating description of neo4j-import data command after incremental import implementation

### PR DESCRIPTION
**Pull request by lidiazuin**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
Looks like you've updated the documentation!

Check out your changes at https://neo4j-docs-operations-664.surge.sh

**Review by renetapopova — CHANGES_REQUESTED** [review]
Thanks, @lidiazuin. I left a suggestion. With the introduction of incremental import, the command can also be used to import data into an existing database.

File: modules/ROOT/pages/tutorial/neo4j-admin-import.adoc [hunk 1]
```diff
@@ -2,11 +2,8 @@
 = Neo4j-admin import
 :description: This tutorial provides detailed examples to illustrate the capabilities of importing data from CSV files with the command `neo4j-admin database import`. 
 
-This tutorial provides detailed examples to illustrate the capabilities of importing data from CSV files with the command `neo4j-admin database import`. 
-
-The xref:tools/neo4j-admin/neo4j-admin-import.adoc[`neo4j-admin database import`] is a command for loading large amounts of data from CSV files into an **unused non-existing** database.
-Importing data from CSV files with `neo4j-admin database import` can only be done once into an **unused** database, it is used for the initial graph population only.
-The `neo4j-admin database import` command can be used on the local Neo4j instance even if the instance is running or not.
+This tutorial provides detailed examples to illustrate the capabilities of importing data from CSV files with the command xref:tools/neo4j-admin/neo4j-admin-import.adoc[`neo4j-admin database import`]. 
+It is used for loading large amounts of data from CSV files and supports full and incremental import into a running or stopped Neo4j DBMS.
```
Comment (renetapopova) [inline modules/ROOT/pages/tutorial/neo4j-admin-import.adoc:6]: "It" refers to "tutorial". I would suggest:
```suggestion
The import command is used for loading large amounts of data from CSV files and supports full and incremental import into a running or stopped Neo4j DBMS.
```

**Review by renetapopova — APPROVED** [review]
Looks good now! Just a small comment. Thank you!

**Pull request merged**